lib/pure/net.nim

```diff
@@ -675,10 +675,10 @@ when defineSsl:
       raiseSSLError()
     when not defined(openssl10) and not defined(libressl):
       let sslVersion = getOpenSSLVersion()
-      if sslVersion >= 0x010101000 and not sslVersion == 0x020000000:
+      if sslVersion >= 0x010101000 and sslVersion != 0x020000000:
         # In OpenSSL >= 1.1.1, TLSv1.3 cipher suites can only be configured via
         # this API.
-        if newCTX.SSL_CTX_set_ciphersuites(cipherList) != 1:
+        if newCTX.SSL_CTX_set_ciphersuites(ciphersuites) != 1:
```

Comment:
I added a new parameter since `ciphersuites` requires at least `CiphersModern`, which is TLS 1.3 settings. It's better to have an Option type, so the check only exists when users specify it. I might document it in the changelog.

See also
https://media.discordapp.net/attachments/371759389889003532/1070559792210386984/image.png